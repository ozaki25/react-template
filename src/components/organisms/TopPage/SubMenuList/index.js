import React from 'react';
import styled from 'styled-components';
import MenuButton from 'src/components/atoms/buttons/MenuButton';

function SubMenuList() {
  return (
    <Container>
      <MenuButtonContainer>
        <MenuButton onClick={() => console.log('click')}>申込履歴の確認</MenuButton>
      </MenuButtonContainer>
      <MenuButtonContainer>
        <MenuButton onClick={() => console.log('click')}>よくあるご質問</MenuButton>
      </MenuButtonContainer>
      <MenuButtonContainer>
        <MenuButton onClick={() => console.log('click')}>残高照会・振込等</MenuButton>
      </MenuButtonContainer>
    </Container>
  );
}

export const Container = styled.ul`
  margin: 0;
  padding: 0;
`;

export const MenuButtonContainer = styled.li`
  margin-bottom: 10px;
  list-style-type: none;
`;

SubMenuList.propTypes = {};

SubMenuList.defaultProps = {};

export default SubMenuList;
