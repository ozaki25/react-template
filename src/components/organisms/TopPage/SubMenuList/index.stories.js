import React from 'react';

import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';

import SubMenuList from '.';

const stories = storiesOf('organisms/TopPage', module);

stories.add('SubMenuList', withInfo('SubMenuList')(() => <SubMenuList />));
