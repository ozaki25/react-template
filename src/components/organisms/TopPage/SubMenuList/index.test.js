import React from 'react';
import { snapshot } from 'test/helpers';
import SubMenuList, { Container, MenuButtonContainer } from '.';

snapshot('SubMenuList', <SubMenuList />);
snapshot('SubMenuList/Container', <Container />);
snapshot('SubMenuList/MenuButtonContainer', <MenuButtonContainer />);
