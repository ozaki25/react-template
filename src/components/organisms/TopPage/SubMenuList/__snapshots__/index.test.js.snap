// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: SubMenuList 1`] = `
<styled.ul>
  <styled.li>
    <MenuButton
      onClick={[Function]}
    >
      申込履歴の確認
    </MenuButton>
  </styled.li>
  <styled.li>
    <MenuButton
      onClick={[Function]}
    >
      よくあるご質問
    </MenuButton>
  </styled.li>
  <styled.li>
    <MenuButton
      onClick={[Function]}
    >
      残高照会・振込等
    </MenuButton>
  </styled.li>
</styled.ul>
`;

exports[`should match snapshot: SubMenuList/Container 1`] = `
.c0 {
  margin: 0;
  padding: 0;
}

<ul
  className="c0"
/>
`;

exports[`should match snapshot: SubMenuList/MenuButtonContainer 1`] = `
.c0 {
  margin-bottom: 10px;
  list-style-type: none;
}

<li
  className="c0"
/>
`;
